components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -60,11 +64,39 @@ - (void)setUpLayoutDirection {
 
 #pragma mark UIView Overrides
 
+- (void)layoutSubviews {
+  [self preLayoutSubviews];
+  [super layoutSubviews];
+  [self postLayoutSubviews];
+}
+
 - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
   [super traitCollectionDidChange:previousTraitCollection];
   [self setUpLayoutDirection];
 }
 
+#pragma mark Layout
+
+- (void)preLayoutSubviews {
+  CGSize fittingSize = CGSizeMake(CGRectGetWidth(self.frame), CGFLOAT_MAX);
```

Comment:
Use `self.bounds` instead of `self.frame` in layout methods for width/height calculations because it is unaffacted by other properties (unlike `self.frame`, which is a composite of self.layer.bounds, self.layer.position, self.layer.transform, and self.layer.anchorPoint).

Reply:
Done.